api/cloudprotocol/unitstatus.go

```diff
@@ -64,6 +69,21 @@ const (
 	NodeStatusError         = "error"
 )
 
+// Node attribute names.
+const (
+	NodeAttrMainNode      = "MainNode"
+	NodeAttrAosComponents = "AosComponents"
+	NodeAttrRunners       = "NodeRunners"
+)
+
+// Node attribute values.
+const (
+	AosComponentCM  = "cm"
```

Comment:
Maybe rename them to AosServices, to be more precise?

Reply:
I'm afraid it's confusing with services we run as containers.